### ISSUE TITLE
[Bug][A11Y][Radio button] Add/modify the hint for unselected radio button

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#469]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

.

### OS Version

Android and iOS

### App version

0.6.0

### Describe the issue

on Android, a hint is present for unselected radio button "double-tap to activate" (not appropriate)

 on iOS: hint is missing

### Expected Behavior

on unselected radio-button, add the hint "Double-tap to toggle " for both apps
